lang/python/2to3/Makefile

```
@@ -0,0 +1,60 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=2to3
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>, Jeffery To <jeffery.to@gmail.com>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_HOST_ONLY:=1
+HOST_BUILD_DEPENDS:=python3/host
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/2to3
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=2to3 binary using python3
+  URL:=https://salsa.debian.org/cpython-team/python3-defaults
+  DEPENDS:= \
+	+python3-lib2to3 \
+	+python3-light
+  BUILDONLY:=1
+endef
+
+define Package/2to3/description
+  2to3 is a Python program that reads Python 2.x source code and applies a
+  series of fixers to transform it into valid Python 3.x code. The standard
+  library contains a rich set of fixers that will handle almost all code. 2to3
+  supporting library lib2to3 is, however, a flexible and generic library, so it
+  is possible to write your own fixers for 2to3. lib2to3 could also be adapted
+  to custom applications in which Python code needs to be edited automatically.
+endef
+
+define Python3/FixShebang
+	$(SED) "1"'!'"b;s,^#"'!'".*python.*,#"'!'"/usr/bin/python3," -i --follow-symlinks $(1)
+endef
+
+define Host/Install
+	$(INSTALL_DIR) $(1)/bin
+	$(INSTALL_BIN) ./files/2to3.py $(1)/bin/2to3
+	if [ -d "$$(1)/bin" ]; then \
+		$$(call Python3/FixShebang,$$(1)/bin/*) ; \
+	fi
```

```suggestion
	$(SED) '1c#!$(STAGING_DIR_HOSTPKG)/bin/python3' $(1)/bin/2to3
```

I haven't tested this though.

---

Okay, I modify this...

---

tested shortly, look like to work
thanks